test/helpers.lua

```diff
@@ -89,6 +89,9 @@ function module.pcall_err(fn, ...)
     error('expected failure, but got success')
   end
   local errmsg = tostring(rv):gsub('^[^:]+:%d+: ', '')
+  errmsg = tostring(errmsg):gsub('^Error executing lua: ', '')
+  errmsg = tostring(errmsg):gsub('^Error loading lua: ', '')
+  errmsg = tostring(errmsg):gsub('^[^:]+:%d+: ', '')
```

Comment:
I really want write like this.
```lua
local errmsg = tostring(rv):gsub('^[^:]+:%d+: (Error executing lua: | Error loading lua: )+([^:]+:%d+: )*', '')
```
But lua does not have full regular expressions.